apps/meteor/server/methods/readThreads.js

```diff
@@ -22,12 +23,15 @@ Meteor.methods({
 		}
 
 		const user = Meteor.user();
+		callbacks.run('beforeReadMessages', thread.rid, user._id);
```

Comment:
same as the other one, this should be moved to right before the call to `readThread()` function

Reply:
Thankss, fixed!